lib/routes/ehentai/ehapi.js

```diff
@@ -23,7 +25,21 @@ function ehgot(url) {
     }
 }
 
-async function parsePage(cache, data, get_bittorrent = false) {
+function ehgot_thumb(cache, thumb_url) {
+    return cache.tryGet(thumb_url, async () => {
+        try {
+            const buffer = await got({ method: 'get', url: thumb_url, headers });
+            const data = new Buffer.from(buffer.rawBody).toString('base64');
+            const ext = path.extname(thumb_url).slice(1);
+            return `data:image/${ext};base64,${data}`;
+        } catch (e) {
+            logger.warn('Cannot download thumbnail: ' + thumb_url, e);
+            return thumb_url;
+        }
+    });
+}
+
+async function parsePage(cache, data, get_bittorrent = false, embed_thumb = false) {
     const $ = cheerio.load(data);
     const table = $('table[class="itg gltc"] tbody');
```

Comment:
The current selector will match "Front Page Settings" of "Compact" only.
```diff
-const table = $('table[class="itg gltc"] tbody');
+const table = $('table[class^="itg glt"] tbody');
```
This will match "Minimal", "Minimal+" and "Compact". It would be better if you can make it match all 5 "Front Page Settings".